otsclient/cmds.py

```diff
@@ -110,6 +139,8 @@ def submit_async_thread(remote, msg, q):
 
 
 def stamp_command(args):
+    install_socks5(args.socks5_proxy)
```

Comment:
`install_socks()` should be elsewhere; stamp is only one of many commands that uses remote calendars. You probably should put that where arguments are parsed instead in `otsclient/args.py` in the `parse_ots_args()` function.